examples/crypto.toit

```diff
@@ -0,0 +1,204 @@
+
+import crypto.aes
```

Comment:
This looks like it should be a test in the `tests/` directory. If you name it <xxx>_test.toit, it should get automatically run when you do 'make test'.

Please also add a copyright/license header.

Reply:
The file has been moved and renamed to crypto_aes_test.toit.